### PR TITLE
zyad contribution

### PR DESCRIPTION
**Pull request by dayZgarden**
Added my own card to index.html

### CODE REVIEW

**Comment by dayZgarden** [comment]
changes requested

File: index.html [hunk 1]
```diff
@@ -142,6 +142,31 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________Famous' card END ________  -->
 
+        <!-- ________ Zyad's card START ________  -->
+        <div class="card">
+          <p class="name">Zyad</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@zdaisygarden</a>
```
Comment (TenTraicion) [inline index.html:150]:
```suggestion
            <a href="https://www.twitter.com/zdaisygarden" target="_blank">@zdaisygarden</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Kindly make the recommended changes!

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@dayZgarden, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**